### PR TITLE
Updating dockerfiles to include lables/licenses for Openshift Container Images

### PR DESCRIPTION
**Pull request by abhis2112**

### CODE REVIEW

File: docker/Dockerfile-gbpserver [hunk 1]
```diff
@@ -1,5 +1,13 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 COPY dist-static/gbpserver /usr/local/bin/
-
+# Required OpenShift Labels
+LABEL name="ACI CNI gbserver" \
```
Comment (jojimt) [inline docker/Dockerfile-gbpserver:4]: Pls change "gbserver" to "gbpserver" everywhere in this file.

File: docker/Dockerfile-operator [hunk 2]
```diff
@@ -1,7 +1,16 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
-RUN yum --disablerepo=\*ubi\* install -y curl git \
-  && yum clean all \
-  && curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.14.6/bin/linux/amd64/kubectl \
-  && chmod u+x kubectl && mv kubectl /usr/local/bin/kubectl
+#RUN yum --disablerepo=\*ubi\* install -y curl git \
```
Comment (jojimt) [inline docker/Dockerfile-operator:2]: Is this commenting intentional?

Comment (abhis2112) [reply docker/Dockerfile-operator:2]: checking....

File: docker/Dockerfile-gbpserver (hunk 1 above)
Comment (abhis2112) [reply docker/Dockerfile-gbpserver:4]: sure...

**Comment by snaiksat** [comment]
Thanks @abhis2112 . Since some of the information here is public facing (via OpenShift operator hub) we should revisit and polish it a little more in a follow up patch.

**Pull request merged**